src/wp-includes/script-loader.php

```diff
@@ -3508,12 +3507,12 @@ function _wp_theme_json_webfonts_handler() {
 				$item['url'] = wp_make_link_relative( $item['url'] );
 			}
 
-			$src .= ( 'data' === $item['format'] )
-				? ", url({$item['url']})"
-				: ", url('{$item['url']}') format('{$item['format']}')";
+			$items[] = ( 'data' === $item['format'] )
+				? "url({$item['url']})"
+				: "url('{$item['url']}') format('{$item['format']}')";
 		}
 
-		return $src;
+		return implode( ', ', $items );
```

Comment:
This change is out-of-scope for the defect bugfix. Let's instead keep parity with the fix made in Gutenberg https://github.com/WordPress/gutenberg/pull/47254.

I'll update this PR.

Reply:
@Luehrsen Though I do understand why this change was made, I've reverted it for the commit.

Why? To keep it matching (parity with) the same code in the Fonts API (being developed in Gutenberg).

This is intentional to keep what's in Core synchronized with what's in the API being developed in Gutenberg.

This change could be considered in the Fonts API. Please feel free to open an [issue](https://github.com/WordPress/gutenberg/issues) and [PR](https://github.com/WordPress/gutenberg/pulls) for discussion. Thank you!